telemetry/definitions/commonDefinitions.json

```diff
@@ -3430,23 +3473,50 @@
             "passive": true
         },
         {
-            "name": "codeTransform_buildComplete",
-            "passive": true,
-            "description": "During the transformation progress, log the status of the build.",
+            "name": "codeTransform_dependenciesCopied",
+            "description": "Deprecated - use codeTransform_uploadProject (The repo copies over at least one dependency successfully)",
             "metadata": [
                 {
-                    "type": "buildDuration",
+                    "type": "codeTransformSessionId",
                     "required": true
                 }
             ]
         },
         {
-            "name": "codeTransform_dependenciesCopied",
-            "description": "The repo copies over at least one dependency successfully.",
+            "name": "codeTransform_downloadArtifact",
+            "description": "Transform or user downloads code transform artifact",
             "metadata": [
+                {
+                    "type": "codeTransformArtifactType",
+                    "required": true
+                },
+                {
+                    "type": "codeTransformJobId",
+                    "required": true
+                },
+                {
+                    "type": "codeTransformRuntimeError",
+                    "required": false
+                },
+                {
+                    "type": "codeTransformRunTimeLatency",
+                    "required": true
+                },
                 {
                     "type": "codeTransformSessionId",
                     "required": true
+                },
+                {
+                    "type": "codeTransformTotalByteSize",
+                    "required": true
+                },
+                {
+                    "type": "reason",
+                    "required": false
+                },
+                {
+                    "type": "result",
+                    "required": true
```

Comment:
please add `passive` if this metric is automatically generated.

Reply:
Right now it's not automatically generated (updated description)